### ISSUE TITLE
Trying out breeze-validation custom attribute

### ISSUE THREAD

**Issue by gregoryagu** [jdanyow/aurelia-breeze#15]
I put together a skeleton project with an MVC/EF backend.

The database has just one table with two fields, FirstName and LastName.  The last name has the `Required` attribute.

I then had the EntityManager fetch the metadata which included a `nullable:false` for LastName which then should be validated in the UI.

It all wired up OK and I can save the contact to the database.
However, when I add "breeze-validation.bind='currentContact'", it throws the following error:

```
TypeError: Cannot read property 'subscribe' of undefined
at BreezeValidation.subscribe (http://localhost:56321/jspm_packages/github/jdanyow/aurelia-breeze@0.11.0/breeze-validation.js:104:70)
```

Stepping through the code in the debugger, it shows that the  subscribe function is expecting and entityManager, but it is getting the entity instead.

This is the line of code that throws an error:
`this.errorsSubscription = entityManager.validationErrorsChanged.subscribe`

This is the viewmodel:

``` javascript

import breeze from "breeze";

export class Welcome {
  currentContact;
  manager;

  constructor() {
      this.manager = new breeze.EntityManager('breeze/data');
      this.manager.fetchMetadata().then(() => {

          this.currentContact = this.manager.createEntity("Contact");
          this.currentContact.FirstName = "Greg";
          this.currentContact.LastName = "Gum";

      });
  }

  submit() {
      this.manager.saveChanges();
  }


} 
```

**Comment by jdanyow** [comment]
Thanks for reporting this, I've reproduced locally and will get a fix in for this issue.  

In the meantime, the following changes to the view-model should resolve the problem:

``` javascript
import breeze from "breeze";

export class Welcome {
  currentContact;
  manager;

  activate() {
      this.manager = new breeze.EntityManager('breeze/data');
      return this.manager.fetchMetadata().then(() => {

          this.currentContact = this.manager.createEntity("Contact");
          this.currentContact.FirstName = "Greg";
          this.currentContact.LastName = "Gum";
      });
  }

  submit() {
      this.manager.saveChanges();
  }
} 
```

keep the feedback/issue-reports coming!

**Comment by jdanyow** [comment]
New release published that should address this issue- v0.11.1

**Comment by gregoryagu** [comment]
I updated to v0.11.2 and tested and it works great!
